landing_page/app/controllers/sessions_controller.rb

```diff
@@ -18,4 +18,7 @@ def destroy
     redirect_to root_url, info: 'Logout Successful!'
   end
 
+  def update 
+  end
```

Comment:
This empty method isn't necessary. Rails handles those by using the "Rendering by Default" approach. You can start reading more about the topic here: https://stackoverflow.com/a/35358871/1456509